inc/class_parser.php

```diff
@@ -1064,7 +1064,11 @@ function mycode_parse_url($url, $name="")
 		$nofollow = '';
 		if(!empty($this->options['nofollow_on']))
 		{
-			$nofollow = " rel=\"nofollow\"";
+			$nofollow = " rel=\"noopener nofollow\"";
+		}
+		else
+		{
+			$nofollow = " rel=\"noopener\"";
```

Comment:
Better to have this in [the template](https://github.com/dvz/mybb/blob/23405ae39351af0a7212c860f496857da9f37f53/install/resources/mybb_theme.xml#L8718) where the `target="_blank"` is to avoid confusion?

Reply:
If we chose to edit the template a separate variable (like `{$rel}`, with hardcoded value) would be a better solution so it doesn't possibly have to be modified again - I didn't go this way yet to make sure want that given recent problems with themes that haven't been updated (and compatibility with e.g. http://community.mybb.com/mods.php?action=view&pid=450 that parses those).

Reply:
Yep as you have done in 03de79e1f99c58857683b6fa531a9e584b0b9674 is fine then :smiley: